kubetest2-gke/deployer/options/cluster.go

```diff
@@ -45,6 +52,8 @@ type ClusterOptions struct {
 	WindowsMachineType string `flag:"~windows-machine-type" desc:"For use with gcloud commands to specify the machine type for Windows node in the cluster."`
 	WindowsImageType   string `flag:"~windows-image-type" desc:"The Windows image type to use for the cluster."`
 
+	ExtraNodePool []string `flag:"~extra-nodepool" desc:"create extra nodepools. options as key=value. supported options are name,machine-type,image-type,num-nodes"`
```

Comment:
are multiple pools specified by repeating the arg or comma-separating values? e.g. `--extra-nodepool ... --extra-nodepool ...` or `--extra-nodepool ...,...`? indicate in the help string which it is

Reply:
updated the help text.

Reply:
I'm not seeing the updated help text indicating how multiple pools are specified... am I missing it?